File: cpp/bc1038.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int codL, qtd;
	float pre;
	
	cin >> codL >> qtd;
	
	switch(codL)
	{
		case 1: pre = 4.00*qtd; break;
		case 2: pre = 4.50*qtd; break;
		case 3: pre = 5.00*qtd; break;
		case 4: pre = 2.00*qtd; break;
		case 5: pre = 1.50*qtd; break;
	}
	
	cout << "Total: R$ " << fixed << setprecision(2) << pre << endl;
}

File: python/bc1038.py
valores = input().split(" ")

cod = int(valores[0])
qtd = int(valores[1])

if cod == 1: total = 4.00*qtd
elif cod == 2: total = 4.50*qtd
elif cod == 3: total = 5.00*qtd
elif cod == 4: total = 2.00*qtd
else: total = 1.50*qtd

print(f"Total: R$ {total:.2f}")